user/storage.md

```diff
@@ -0,0 +1,266 @@
+# Files and Data in the Cloud
+
+This page describes how files and data storage are handled in 2i2c Hubs.
+The high-level summary of recommendations is:
+- Use your home directory to store code, notebooks, and small data files (<1 GB)
+  for personal use
+- Use cloud object storage to store larger datasets and to share data across your team
+- Consider whether your project would benefit from other cloud-native data storage
+  solutions such as a database, data warehouse, or data lake
+
+:::{admonition} Attribution
+The following material was adapted from the
+[Pangeo Cloud User Guide](https://pangeo.io/cloud.html)
+:::
+
+## Your Home Directory
+
+Your notebook server is a linux "virtual machine" with its own filesystem.
+Your are not on a shared server; you are on your own private server.
+Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
+This is the same for all users.
+
+Your home directory is intended only for notebooks, analysis scripts, and small datasets (< 1 GB).
+It is not an appropriate place to store large datasets.
+No one else can see or access the files your home directory.
+
+The easiest way to move files in and out of your home directory is via the JupyterLab web interface.
+Drag a file into the file browser to upload, and right-click to download back out.
+You can also open a terminal via the JupyterLab launcher and use this to ssh / scp / ftp to remote systems.
+However, you can’t ssh in!
```

Comment:
Is it true that you can't ssh / scp into your server? Or do we have a solution for that? If so, this would be a good place to mention it.

Reply:
https://github.com/yuvipanda/jupyterhub-ssh adds SSH and SFTP (but not scp) support. Maybe we could make that an additional feature we can deploy. I know that @consideRatio has this going for @fperez's JMTE hub for a while - how has your experience been, @consideRatio?

Reply:
I think we've had a few surprises learning about some limitations, but have been happy about what is supported.

- `rsync` over SSH -> SFTP recommended to use instead -> Some confusion about that we had to use another port than port 22 for that as we had to distinguish the JupyterHub SSH server from the JupyterHub SFTP server.
  https://github.com/yuvipanda/jupyterhub-ssh/issues/55
- Some challenges about servers being culled by `jupyterhub-idle-culler` when having an open SSH connection
  https://github.com/yuvipanda/jupyterhub-ssh/issues/67
- SSH Tunnels aren't supported, but perhaps an expectation held for someone installing `jupyterhub-ssh`
  https://github.com/yuvipanda/jupyterhub-ssh/issues/26

Reply:
Let's leave ssh out of this guide for now. We can always update it later.

I also think it's fine if we try to push people towards a more cloud-native approach to data.